### PR TITLE
fix detach and add docs

### PR DESCRIPTION
**Pull request by livingshade**
## Why are these changes needed?

Current detach.toml for hotel reservation is incorrect.

There also lacks document for attach/detach.toml, I have added some based on my understanding. 

## Checks

- [✅] I've run `scripts/clippy.sh` to lint the changes in this PR.
- [✅] I've included any doc changes.
- Testing Strategy
   - [ ] Release tests
   - [✅] This PR is not tested :(

### CODE REVIEW

File: mdbook/src/tutorials/policy-management.md [hunk 1]
```diff
@@ -96,3 +96,33 @@ After the following command is executed, rate limit policy is no longer applied.
 ```
 cargo run  --release --bin addonctl -- --config eval/policy/ratelimit/detach.toml --pid 2012290 --sid 1
 ```
+
+# Semantics
+
+The engines can form a graph, and is connected via tx/rx channels, which are bidirectional.
```
Comment (crazyboycjr) [inline mdbook/src/tutorials/policy-management.md:102]: Do you mean tx+rx are bidirectional?

Comment (crazyboycjr) [reply mdbook/src/tutorials/policy-management.md:102]: Maybe `The engines can form a graph, and are connected via unidirectional tx/rx channels.`?

Comment (livingshade) [reply mdbook/src/tutorials/policy-management.md]: Sure, I will fix that.

**Review by crazyboycjr — APPROVED** [review]
LGTM. Thanks so much for the doc!

**Pull request merged**